legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/runtime/connection/datasourceSpecification.pure

```diff
@@ -60,8 +60,25 @@ Class meta::pure::alloy::connections::alloy::specification::SnowflakeDatasourceS
     quotedIdentifiersIgnoreCase:Boolean[0..1];
 }
 
+Enum meta::pure::alloy::connections::alloy::specification::Region
```

Comment:
Haha .. this is my point :) This list is missing the AWS milan region https://aws.amazon.com/about-aws/whats-new/2020/04/announcing-the-new-aws-europe-milan-region/